docs/resources/built-in-rules.md

```diff
@@ -328,3 +328,225 @@ Here is the equivalent of the `recommended` configuration values:
 
 OpenAPI-cli supports [custom rules](./custom-rules.md).
 However, if you have an idea for a built-in rule you believe will benefit the greater API community, please [open an issue](https://github.com/Redocly/openapi-cli/issues/new).
+
+## Generic built-in rules
+
+You can use special `enforcements` rule for cases which our other built-in rules do not cover.
+There are [a set of generic built-in rules](#list-of-generic-built-in-rules) which can be configured under `enforcements` to enforce different generic usecases.
+
+### How it works?
+
+Let's say you want to validate Operation summary and Info description fields - they should be defined, not empty, 
+minimum 30 chars length and both have "_hello_" substring inside.
+For these usecases you can create next enforcements:
+
+```yaml
+lint:
+  rules:
+    enforcements:
+      - on
+        - Operation.summary
+        - Info.description
+        description: Summary and Description fields don't match enforcements
+        severity: error
+        defined: true
+        nonEmpty: true
+        minLength: 30
+        regex: /hello/
+```
+
+In other words, it generates 8 unique custom rules `operation-summary-defined`, `operation-summary-non-empty`, `operation-summary-min-length`, `operation-summary-regex`,
+`info-description-defined`, `info-description-non-empty`, `info-description-min-length`, `info-description-regex`.
```

Comment:
This part is confusing. It doesn't generate 8 unique custom rules technically. Also what are those names? We don't have such rules. I would remove it.